dotCMS/src/main/java/com/dotmarketing/util/FileUtil.java

```diff
@@ -39,6 +39,13 @@ public static File createTemporaryFile(final String prefix) throws IOException {
 		return createTemporaryFile(prefix, null);
 	}
 
+	public static File createTemporaryDirectory(final String prefix) throws IOException {
```

Comment:
Is it important that this directory is in java.io.tmpdir and the bundles end up in the `/assets` dir, which is often on another mounted disk such as NFS?

Moving a file from one mount to another is a slow and expensive operation.  I believe we have a `tmp` dir in the `/assets` directory that we might want to use.